scripts/platforms/greenhouse_apply_v2.py

```
@@ -0,0 +1,550 @@
+                context_info = await extract_field_context(checkbox)
+                combined = normalize_key(context_info["combined"])
+                if not combined:
+                    continue
+
+                # Auto-check terms/privacy/consent/acknowledge
+                if any(token in combined for token in ["terms", "privacy", "consent", "certify", "acknowledge"]):
+                    await checkbox.check()
+                    field_memory[combined] = "checked"
+
+            except Exception:
+                continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)